docs/chainlink-keepers/register-upkeep.md

```diff
@@ -20,70 +18,71 @@ This guide explains how to register a Keeper-compatible contract with the Chainl
 
 1. **Click `Register upkeep`** and confirm the transaction in MetaMask
-  This sends a request to the Chainlink Keeper Network that will need to be manually approved. On testnets, requests are automatically approved.
+   This sends a request to the Chainlink Keeper Network that will need to be manually approved. On testnets, requests are automatically approved.
 
-    ![Upkeep Registration Success Message](/images/contract-devs/keeper/keeper-registration-submitted.png)
+   ![Upkeep Registration Success Message](/images/contract-devs/keeper/keeper-registration-submitted.png)
 
 After you complete registration, your upkeep will start being serviced after a predefined block confirmation time, which is less than 10 minutes. The number of block confirmations might differ depending on which blockchain you selected.
 
 ## Fund Upkeep
 
 After registration, you have to monitor the balance of your Upkeep. If the balance runs out then the Keepers network will not perform the Upkeep. Follow these steps to fund your Upkeep.
 
-  1. **Click `View Upkeep`** or go to the [Chainlink Keepers App](https://keepers.chain.link) and click on your recently registered Upkeep
-  
-  1. **Click the `Add funds` button**
-  
-  1. **Approve the LINK spend allowance** 
-    ![Approve LINK Spend Allowance](/images/contract-devs/keeper/keeper-approve-allowance.png)
-  
-  1. **Confirm the LINK transfer** by sending funds to the Chainlink Keeper Network Registry
-    ![Confirm LINK Transfer](/images/contract-devs/keeper/keeper-confirm-transfer.png)
-    
-  1. **Receive a success message** and verify that the funds were added to the Upkeep
-    ![Funds Added Successful Message](/images/contract-devs/keeper/keeper-add-funds.png)
+1. **Click `View Upkeep`** or go to the [Chainlink Keepers App](https://keepers.chain.link) and click on your recently registered Upkeep
+
+1. **Click the `Add funds` button**
+
+1. **Approve the LINK spend allowance**
+   ![Approve LINK Spend Allowance](/images/contract-devs/keeper/keeper-approve-allowance.png)
+
+1. **Confirm the LINK transfer** by sending funds to the Chainlink Keeper Network Registry
+   ![Confirm LINK Transfer](/images/contract-devs/keeper/keeper-confirm-transfer.png)
+
+1. **Receive a success message** and verify that the funds were added to the Upkeep
+   ![Funds Added Successful Message](/images/contract-devs/keeper/keeper-add-funds.png)
 
 ## How Funding Works
 
-* Your balance is reduced each time a Keeper executes your `performUpkeep` method.
-* There is no cost for `checkUpkeep` calls.
-* If you want to automate adding funds, you can directly call the `addFunds()` function on the `KeeperRegistry` contract.
-* Anyone can call the `addFunds()` function, not just the Upkeep owner.
-* To withdraw funds, cancel the Upkeep and then click **Withdraw funds**.
+- Your balance is reduced each time a Keeper executes your `performUpkeep` method.
+- There is no cost for `checkUpkeep` calls.
```

Comment:
I don't want to call this out ("There is no cost for 'checkUpkeep' calls") please check with me for details

Reply:
@ritika-b @wentzeld : can you please discuss this point during your meeting? (cf. https://docs.chain.link/docs/chainlink-keepers/keeper-economics/#how-funding-works)